### DISCUSSION TITLE
need option of 2 years recurring for DSC task

### DISCUSSION THREAD

**Discussion by VRJNCO** [madrecha/jamku#1036] [QA] [answered]
need option of 2 years recurring for DSC task. or is there a better way to track DSC expiry?

**Comment by adarshmadrecha** [comment, marked as answer]
For DSC we don't recommend creating recurring service.
In this case, create a Non-Recurring Service and create tasks as and when required.

Reasons:
1. There can be more than ONE DSC tasks for a client. Example: In case of PVT LTD company, DSC tasks are created for all the directors. If "recurring" service is used it will not allow to create more than one task. In case of NON-Recurring, it will allow this.
1. DSC Issuing does have a fix due date. It will vary client to client
2. All the DSC tasks are not created in ONE GO. They are created one by one.
3. Case to case basis you may apply for DSC for one year or two years.

**Reply by VRJNCO** [reply]
makes sense. noted with thanks.
